helpers.js

```diff
@@ -11,7 +11,7 @@ const helpersList = [
     'dynamicComponent',
     'encodeHtmlEntities',
     'for',
-    'get',
+    // 'get',
```

Comment:
Lets maybe enable 1 helper here? Maybe `get`

Reply:
Let's start with `moment` and `option`.